web/annotation_editor_layer_builder.css

```diff
@@ -15,17 +15,22 @@
 
 :root {
   --outline-width: 2px;
-  --outline-color: blue;
+  --outline-color: #0060df;
+  --outline-around-width: 1px;
+  --outline-around-color: #f0f0f4;
   --focus-outline: solid var(--outline-width) var(--outline-color);
+  --unfocus-outline: solid var(--outline-width) transparent;
+  --focus-outline-around: solid var(--outline-around-width)
+    var(--outline-around-color);
   --hover-outline: dashed var(--outline-width) var(--outline-color);
   --freetext-line-height: 1.35;
   --freetext-padding: 2px;
-  --resizer-size: 8px;
+  --resizer-size: 6px;
   --resizer-shift: calc(
-    0px - var(--outline-width) - var(--resizer-size) / 2 - var(--outline-width) /
-      2
+    0px - (var(--outline-width) + var(--resizer-size)) / 2 -
+      var(--outline-around-width)
   );
-  --resizer-color: white;
+  --resizer-color: var(--outline-color);
```

Comment:
Do we still need this variable, or could we simply change [this line](https://github.com/mozilla/pdf.js/pull/16943/files#diff-52f5d77845bc000fab73ee0756ed15ce132e88c1893bff5a2268402ea99599aaL201) into `background: var(--outline-color);` instead?